### PR TITLE
Update CONTRIBUTING to be not so outdated

### LINKED ISSUE #87: Contributing guide does not include Library Carpentry
Whereas the README file links to Software, Data and Library Carpentry, the Contributing Guide only includes links to Software and Data Carpentry. Other contributing guides also include links to The Carpentries website.

I did update several links in this guide in #85 but didn't update these references yet.

### PR DESCRIPTION
**Pull request by zkamvar**
This brings in the changes from the workbench template repo

This will fix #87

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,134 +1,123 @@
...
 
+[repo]: https://github.com/carpentries/maintainer-onboarding/
+[repo-issues]: https://github.com/carpentries/maintainer-onboarding/issues
 [contact]: mailto:team@carpentries.org
+[cp-site]: https://carpentries.org/
 [dc-issues]: https://github.com/issues?q=user%3Adatacarpentry
 [dc-lessons]: https://datacarpentry.org/lessons/
 [dc-site]: https://datacarpentry.org/
-[discuss-list]: https://carpentries.topicbox.com/groups/discuss
+[discuss-list]: https://lists.software-carpentry.org/listinfo/discuss
```
Comment (zkamvar) [inline CONTRIBUTING.md:112]: Oh that's embarassing

```suggestion
[discuss-list]: hhttps://carpentries.topicbox.com/groups/discuss
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,134 +1,123 @@
...
 
+[repo]: https://github.com/carpentries/maintainer-onboarding/
+[repo-issues]: https://github.com/carpentries/maintainer-onboarding/issues
 [contact]: mailto:team@carpentries.org
+[cp-site]: https://carpentries.org/
 [dc-issues]: https://github.com/issues?q=user%3Adatacarpentry
 [dc-lessons]: https://datacarpentry.org/lessons/
 [dc-site]: https://datacarpentry.org/
-[discuss-list]: https://carpentries.topicbox.com/groups/discuss
+[discuss-list]: hhttps://carpentries.topicbox.com/groups/discuss
```
Comment (zkamvar) [inline CONTRIBUTING.md:112]: 😩 

```suggestion
[discuss-list]: https://carpentries.topicbox.com/groups/discuss
```

Comment (bencomp) [reply CONTRIBUTING.md]: Relax! 😎

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -125,10 +114,10 @@ You can also [reach us by email][contact].
 [github-flow]: https://guides.github.com/introduction/flow/
 [github-join]: https://github.com/join
 [how-contribute]: https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github
-[issues]: https://guides.github.com/features/issues/
-[repo]: https://github.com/carpentries/maintainer-onboarding/
-[repo-issues]: https://github.com/carpentries/maintainer-onboarding/issues/
+[issues]: https://carpentries.org/help-wanted-issues/
```
Comment (bencomp) [inline CONTRIBUTING.md:117]: In the text, it isn't clear to me that "submit bug reports" would take me to the list of help-wanted issues. It used to go to GitHub help on issues. Is this intentional?

**Review by bencomp — COMMENTED** [review]
I wonder if the headers should start at level 1 instead of 2?

**Pull request merged**